src/offices/_single.html

```diff
@@ -26,12 +28,14 @@
         {% set show_subscription = intro.subscribe_form and intro.govdelivery_code %}
         <div class="content-l_col
                         {% if show_subscription -%}
-                            content-l_col-1-2
+                        content-l_col-1-2
                         {% else -%}
-                            content-l_col-1
+                        content-l_col-1
                         {%- endif %}
                         ">
-            <p class="h3">{{ intro.text | safe if intro.text }}</p>
+            {# Note this is a div to accommodate paragraph markup
+               coming from the CMS for the intro.text content. #}
+            <div class="h3">{{ intro.text | safe }}</div>
```

Comment:
Because in some cases it looks like:

```
Some text.
```

Fine, great. Then other times it looks like:

```
<p>Some text</p>
```

Hmm, okay, we could still strip the tags and keep the enclosing paragraph element.

Then other times:

```
<p>Some text</p> 
<ul>
   <li>Oh, let's put a bulleted list under it.</li>
   <li>Oh you like a bulleted list?</li>
   <li>Goooooood... because it gets better...</li>
</ul>
```

Arrrrrgh! I guess we can't use the enclosing p element anymore or reliably strip tags... and um...

```
<p>Some text.</p>
<p>Some more text</p>
<ul> <li>Some more bullets</li> </ul>
<p><a href="#">How about a link, why not?</a></p>
```